starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/ListAdapter.java

```diff
@@ -0,0 +1,41 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.List;
+
+/**
+ * Created by jay on 10/25/16.
+ */
+
+public class ListAdapter extends RecyclerView.Adapter<ListHolder> {
```

Comment:
As with the ItemObject class, I suggest using a more descriptive name for this class, like "ShoppingRecyclerViewAdapter" or "GroceryRvAdapter"